### PR TITLE
Bug/ag grid gets cut

### PR DESCRIPTION
**Pull request by qoobaa**
screenshot 2017-11-0218 53 28

It doesn't look so great anymore, but it works.

### CODE REVIEW

File: app/assets/javascripts/views/table_view.js [hunk 1]
```diff
@@ -38,26 +37,18 @@
...
         rowHeight: 60,
         colWidth: 150,
+        domLayout: 'autoHeight',
         icons: {
           sortUnSort: '<svg class="icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-short"></use></svg>',
           sortAscending: '<svg class="icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-short"></use></svg>',
           sortDescending: '<svg class="icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-short"></use></svg>'
         }
       };
+      this.$el.wrap('<div class="row">');
```
Comment (qoobaa) [inline app/assets/javascripts/views/table_view.js]: This should be moved to ERB-s.

**Comment by simaob** [comment]
Glad I can see the whole data =D Maybe adding a bit more space on the left would be good, so that the country name has a bit more space.

**Comment by qoobaa** [comment]
I think the inside of the grid is managed automatically by ag-grid (it doesn't mean that we can't change it, just not sure if it's worth doing it).

**Comment by simaob** [comment]
OK, let's leave it as is then. Not worth dwelling too much now.

**Pull request merged**